src/app/tests/TestTimedHandler.cpp

```diff
@@ -93,189 +115,153 @@ class TestExchangeDelegate : public Messaging::ExchangeDelegate
 
 } // anonymous namespace
 
-void TestTimedHandler::GenerateTimedRequest(nlTestSuite * aSuite, uint16_t aTimeoutValue, System::PacketBufferHandle & aPayload)
+void TestTimedHandler::GenerateTimedRequest(uint16_t aTimeoutValue, System::PacketBufferHandle & aPayload)
 {
     aPayload = System::PacketBufferHandle::New(System::PacketBuffer::kMaxSize);
-    NL_TEST_ASSERT(aSuite, !aPayload.IsNull());
+    ASSERT_FALSE(aPayload.IsNull());
 
     System::PacketBufferTLVWriter writer;
     writer.Init(std::move(aPayload));
 
     TimedRequestMessage::Builder builder;
     CHIP_ERROR err = builder.Init(&writer);
-    NL_TEST_ASSERT(aSuite, err == CHIP_NO_ERROR);
+    EXPECT_EQ(err, CHIP_NO_ERROR);
```

Comment:
In the other files, there is still a separate assignment and verification of the error, please fix that for the rest of the cases.

``` cpp
    EXPECT_EQ(builder.Init(&writer), CHIP_NO_ERROR);
```

Reply:
done for all the files in this PR, 
FYI, there are many other converted tests where this was not done